UserDocs.md

```diff
@@ -353,3 +353,8 @@ T4 automatically populates the following metadata:
     s3:GetObject
     s3:GetObjectVersion
     ```
+* The keys of objects in S3 should not end in `/`. Objects that end in `/`
+are treated specially by some S3 tools in a way that
+is potentially dangerous, so it's best to avoid them.
+The helium API will help you avoid this rough edge by rejecting keys that end in `/`.
```

Comment:
"by rejecting object keys that end in `/`"
(i.e. folder keys that end in `/` are legit and we should probably spell that out)